hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainerMetadataInspector.java

```diff
@@ -181,55 +182,61 @@ public String process(ContainerData containerData, DatanodeStore store,
       return null;
     }
 
-    JsonObject containerJson = inspectContainer(kvData, store);
-    boolean correct = checkAndRepair(containerJson, kvData, store);
-
-    Gson gson = new GsonBuilder()
-        .setPrettyPrinting()
-        .serializeNulls()
-        .create();
-    String jsonReport = gson.toJson(containerJson);
-    if (log != null) {
-      if (correct) {
-        log.trace(jsonReport);
-      } else {
-        log.error(jsonReport);
+    ObjectNode containerJson = inspectContainer(kvData, store);
+    boolean correct = checkAndRepair(containerJson, kvData,
+        store);
```

Comment:
nit: please avoid unrelated formatting change

```suggestion
    boolean correct = checkAndRepair(containerJson, kvData, store);
```

Reply:
Done!